### PR TITLE
Remove rules

### PR DESCRIPTION
**Pull request by madsnorgaard**
This PR is blocked by https://github.com/OS2Forms/os2forms_forloeb/pull/89

We will need to alter the version to the newest release of the Forløbs module:
https://github.com/OS2Forms/os2forms_forloeb_profile/pull/24/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R22

This line should be updated to the newest release tag of the module once the [PR](https://github.com/OS2Forms/os2forms_forloeb/pull/89) is merged

### CODE REVIEW

**Review by agger-magenta — APPROVED** [review]

**Comment by lats1** [comment]
I get that the main thing is about the rules dependency. But isn't there something else going on in the PR?

**Comment by madsnorgaard** [comment]
@lats1 True, I have looked over the changes which are from a previous commit as the branch for this PR was made from main branch. Long story short these changes are present in Main branch.

We should set develop as default branch I reckon.

File: composer.json [hunk 1]
```diff
@@ -16,10 +16,10 @@
         }
     },
     "require": {
-        "drupal/bootstrap": "^3.23",
+        "drupal/gin": "^3.0",
         "drupal/core": "^8",
         "os2forms/os2forms": "^2.6",
-        "os2forms/os2forms_forloeb": "^1.3",
+        "os2forms/os2forms_forloeb": "dev-remove_rules_modules_from_forloeb",
```
Comment (lats1) [inline composer.json:22]: I don't think a specific branch of the module should be here. It should only be a version constraint similar to the one used before.

**Review by lats1 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (madsnorgaard) [reply composer.json:22]: Making this change straight away as the blocking PR is now merged.

**Comment by madsnorgaard** [comment]
Change as per request to the latest version of the Forløb module with Rules removed as dependency.

**Review by lats1 — APPROVED** [review]

**Pull request merged**